airflow/contrib/operators/gcs_to_bq.py

```diff
@@ -125,6 +130,7 @@ def __init__(
         # GCS config
         self.bucket = bucket
         self.source_objects = source_objects
+        self.schema_bucket = schema_bucket or bucket
```

Comment:
I know this is an `or` property, but wouldn't it be more explicit to use something like

`self.schema_bucket = schema_bucket if schema_bucket is not None else bucket`

Reply:
Yes, probably will cause less confusion.